org-roam-utils.el

```diff
@@ -87,13 +87,14 @@
                                 (file-truename (buffer-file-name (current-buffer))))))
             (list :from file-path
                   :to (file-truename (expand-file-name path (file-name-directory file-path)))
-                  :content content)))))))
+                  :content content
```

Comment:
Instead of this, change `:content` to something more generic like `:properties`, which contains a `plist`:

```
(:content content
 :point p)
```

`p` can be obtained from the org-element properties (`:content-begin` or `:begin`), but we need to be careful about this as this needs to match the actual preview content.

Reply:
> needs to match the actual preview content.

You may change this if you wish if it's reasonable. The worry is pulling out headers (especially level 1 headers), which would mess with the formatting of the buffer.